airflow/utils/dag_processing.py

```diff
@@ -47,6 +47,9 @@
 from airflow.utils.db import provide_session
 from airflow.utils.log.logging_mixin import LoggingMixin
 
+if six.PY2:
```

Comment:
It's good for testing it now, but maybe we should just add it in the cherry-picked version in v1-10-test/stable.

Reply:
Oh yeah, this was cos I started against the release branch then ported it forward. Will remove from this branch.